lib/ui/components/captured-requests.js

```diff
@@ -48,15 +49,18 @@ export class CapturedRequests extends React.Component {
   }
 
   _requestDetailsFor(request) {
+    let mockedState;
+
     if (this._isAlreadyMocked(request)) {
       const mockedRequest = this._getMockedRequest(request);
 
-      return mockedRequest.states
+      mockedState = mockedRequest.states
         .filter((state) => state.id === mockedRequest.selectedStateId)[0]
         .name;
     }
 
-    return <Url url={ request.url }/>;
+    return <Url url={ request.url }
+                mockedState={ mockedState }/>;
```

Comment:
mockedState should not be a part of url component, it is only for url string presentation (truncation and colored parameters)